src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +14,52 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Transaction transaction = null;
+        try (Session session = factory.openSession()) {
+            transaction = session.beginTransaction();
+            session.save(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can`t save Comment " + entity);
+        }
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            Comment comment = null;
+            comment = session.get(Comment.class, id);
+            return comment;
+        } catch (Exception e) {
+            throw new RuntimeException("Can`t get Comment by id " + id);
```

Comment:
I think it's may be better to throw cause of Exception "e" after exception message.

Reply:
++